pallets/process-validation/src/lib.rs

```diff
@@ -79,50 +100,113 @@ pub mod pallet {
 
     /// Storage map definition
     #[pallet::storage]
-    #[pallet::getter(fn processes_by_id_and_version)]
-    pub(super) type ProcessesByIdAndVersion<T: Config> = StorageDoubleMap<
+    #[pallet::getter(fn get_process)]
+    pub(super) type GetProcess<T: Config> = StorageDoubleMap<
         _,
         Blake2_128Concat,
         T::ProcessIdentifier,
         Blake2_128Concat,
-        T::ProcessVersion,
+        i32,
         Process,
+        ValueQuery,
+    >;
```

Comment:
can we remove the `ProcessVersion` type? or is that preferred over `i32`?

Reply:
No, this is something we need to implement. The idea is to have either a global trait in terms of process_validation scope

Reply:
The idea @danielcooperxyz behind having it as a trait is to force consistency of types between versions referenced in the `pallet-simple-nft` `run_procss` extrinsic and the type used in this pallet